manifests/v/vim/vim/8.2.3187/vim.vim.installer.yaml

```
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: vim.vim
+PackageVersion: 8.2.3187
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3187/gvim_8.2.3187_x64.exe
+  InstallerSha256: 1C2368FCE9EC4E227CE438F85FC5AF9551F524609B24C81790B973A78388F9FB
```

Signed version just came out.

```suggestion
  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3187/gvim_8.2.3187_x64_signed.exe
  InstallerSha256: 8702BBD2183FDCFC7351BC32A8E2976229AD60971D4ADFAC1E71FA63D2F0632C
```